src/Sylius/Bundle/ApiBundle/ApiPlatform/Symfony/EventListener/WriteListener.php

```diff
@@ -41,9 +41,6 @@
  * The fix has been merged in PR #6367 (https://github.com/api-platform/core/pull/6367), and we are waiting for the new version of ApiPlatform to be released.
```

Comment:
Given this comment, it's possible we can just remove this whole class.

Reply:
If we bump APIP to version 3.3.5, we can remove this class.

Reply:
https://github.com/Sylius/Sylius/actions/runs/9383182395/job/25836332303#step:9:288

Removed the class

Reply:
I would prefer to lock the minimum version to 3.3.4 to ensure that the adjustments will work.

Reply:
From what I see the newest is 3.3.5, we can just set that as the minimum